index.html

```diff
@@ -8,6 +8,7 @@
     </head>
```

Comment:
Consider adding a semicolon at the end of the new paragraph sentence for consistency.